e2e/features/environment.py

```
@@ -0,0 +1,190 @@
+def after_feature(context: Context, feature: Feature) -> None:
+    """Run after each feature file is exercised."""
+    if "Authorized" in feature.tags:
+        switch_config(context.default_config_backup)
+        restart_container("lightspeed-stack")
+        remove_config_backup(context.default_config_backup)
+
+    if "Feedback" in feature.tags:
+        for conversation_id in context.feedback_conversations:
+            url = f"http://{context.hostname}:{context.port}/v1/conversations/{conversation_id}"
+            headers = context.auth_headers if hasattr(context, "auth_headers") else {}
+            response = requests.delete(url, headers=headers)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)